tests/system/test_aggregation_query.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2011 Google LLC
```

Comment:
let's add some more test cases using count with an existing query (like a filter or something similar)

Reply:
Thanks, it's a great idea. I've added some more tests where the nested query are filtered.

Reply:
Great! I also just noticed the year on this needs updating :)

Reply:
Updated.